parts/k8s/addons/kubernetesmasteraddons-kube-dns-deployment.yaml

```diff
@@ -146,6 +146,8 @@ spec:
         - mountPath: /kube-dns-config
           name: kube-dns-config
       - args:
```

Comment:
healthz container should be removed, as sidecar container has been added below.

Reply:
Also deployed with https://github.com/kubernetes/kubernetes/blob/v1.11.0/cluster/addons/dns/kube-dns/kube-dns.yaml.base, it works well with acs-engine deployed cluster (v1.11.0).